src/ui/common/src/components/integrations/addIntegrations.tsx

```diff
@@ -125,10 +127,20 @@ const AddIntegrations: React.FC<Props> = ({
                     {`Successfully connected to ${service}!`}
                   </Alert>
                 </Snackbar>
+                
               </Grid>
             );
           })}
       </Grid>
+      { 
+        <Alert
```

Comment:
@agiron123 how do i get this to show up at the top of the page instead of as part of this box component

Reply:
See if you can move it above the grid like so:
```
  const [showMigrationDialog, setShowMigrationDialog] = useState(false);

  return (
    <Box sx={{ maxWidth: '616px' }}>
      {showMigrationDialog && (
        <Alert
          onClose={() => {
            setShowMigrationDialog(false);
          }}
          severity="info"
          sx={{ width: '100%' }}
        >
          {`Storage migration is in progress. The server will be temporarily unavailable. Please refresh the page to check if the server is ready.`}
        </Alert>
      )}
      <Grid
        container
        spacing={1}
        sx={{ my: '16px', width: '100%' }}
        columns={4}
      >
```

Reply:
Made the change